README.md

```diff
@@ -88,6 +83,9 @@ customize it to include the relevant Test Kits and content for your platform.
 * This will allow you to rebuild using Jekyll locally, and serve a copy of the static files
 * You can run `bundle exec rake web:serve` to generate the files and view them at `http://localhost:4000`
```

Comment:
You need to run `bundle exec rake db:migrate` before web:serve or it won't work

Reply:
Fixed!  Also need to run `bundle install` or `bundle` so I added that.  Thanks for a close read of the docs.